## HYPOTHESIS TESTING

### State the Hypotheses Statement

#### Null Hypothesis (𝐻0): The true mean weekly operating cost is equal to the theoretical cost given by the model 𝜇0 = 1000 + 5𝑋 μ0, where 𝑋=600 units.
#### Alternative Hypothesis (𝐻𝑎): The true mean weekly operating cost is higher than the theoretical cost, i.e., 𝜇 > 𝜇0

## Calculate the Test Statistic and Determine the Critical Value

In [5]:
import math
from scipy.stats import norm

# Given data
x_bar = 3050  # sample mean
mu = 4000  # theoretical mean weekly cost
sigma = 5 * 25  # standard deviation
n = 25  # sample size
alpha = 0.05  # significance level

# Calculate the test statistic (t)
t_stat = (x_bar - mu) / (sigma / math.sqrt(n))

# Determine the critical value (Z-critical) for alpha = 0.05 (one-tailed test)
z_critical = norm.ppf(1 - alpha)

# Print the test statistic and critical value
print(f"Test Statistic (t): {t_stat}")
print(f"Critical Value (Z): {z_critical}")

Test Statistic (t): -38.0
Critical Value (Z): 1.6448536269514722


## Make a Decision

## Compare the test statistic t with the critical value 1.645. If t>1.645, reject the null hypothesis.

## Conclusion

### If the test statistic exceeds the critical value, it suggests that the weekly operating costs are significantly higher than the model prediction.